ports/fftw3/portfile.cmake

```
@@ -27,12 +28,17 @@ vcpkg_check_features(
         avx     ENABLE_AVX
         sse2    ENABLE_SSE2
         sse     ENABLE_SSE
+        neon    ENABLE_NEON
 )
 
 set(package_names  fftw3 fftw3f fftw3l)
-set(fftw3_options  "")
+if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm32")
+    set(fftw3_options "-DENABLE_NEON=OFF") # neon for double precision is not supported on arm32
+else()
+    set(fftw3_options  "")
+endif()
 set(fftw3f_options -DENABLE_FLOAT=ON)
-set(fftw3l_options -DENABLE_LONG_DOUBLE=ON -DENABLE_AVX2=OFF -DENABLE_AVX=OFF -DENABLE_SSE2=OFF)
+set(fftw3l_options -DENABLE_LONG_DOUBLE=ON -DENABLE_AVX2=OFF -DENABLE_AVX=OFF -DENABLE_SSE2=OFF -DENABLE_NEON=OFF)

```

and here ...  but the add the option as needed, in a single block.
```suggestion
set(fftw3l_options -DENABLE_LONG_DOUBLE=ON -DENABLE_AVX2=OFF -DENABLE_AVX=OFF -DENABLE_SSE2=OFF)

if("neon" IN_LIST FEATURES)
    list(APPEND fftw3l_options -DENABLE_NEON=ON)
    if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
        list(APPEND fftw3_options -DENABLE_NEON=ON)
    endif()
endif()
```